tensorflow/lite/python/lite.py

```diff
@@ -291,7 +291,7 @@ def convert(self):
 class TFLiteConverter(object):
   """Convert a TensorFlow model into `output_format` using TOCO.
 
-  This is used to convert from a TensorFlow GraphDef or SavedModel into either a
+  This is used to convert from a TensorFlow GraphDef, SavedModel or tf.keras model into either a
```

Comment:
nit: Can you fix the formatting (i.e. new line is within 80 characters).

Reply:
Updated new line to be within 80 characters. Thanks for your review.